### PR TITLE
Add reloading after confirm changes for page

### PR DESCRIPTION
**Pull request by fkroockmann**

### CODE REVIEW

File: src/tb/apps/page/controllers/main.controller.js [hunk 1]
```diff
@@ -197,6 +197,9 @@ define(
                                 if (data.length > 0) {
                                     SaveManager.save(data, page.uid).done(function () {
                                         notify.success(translator.translate('page_modification_validated'));
+
+                                        location.reload();
```
Comment (mickaelandrieu) [inline src/tb/apps/page/controllers/main.controller.js]: if you reload the DOM, the code after reloading is executed ?

Comment (fkroockmann) [inline src/tb/apps/page/controllers/main.controller.js]: yes :)

**Comment by fkroockmann** [comment]
Thanks for review :)

**Comment by mickaelandrieu** [comment]
don't merge your owns pull requests please, let @ndufreche or @hbaptiste put a :+1: and merge before :)

**Pull request merged**